CMSgov/ato-blueprint
# Feature/help pages & importing files from branding repo

Story: https://jiraent.cms.gov/browse/ISPGBSS-731
Figma: https://www.figma.com/file/Q0BD09RJhn8TMdI8BwhiSS/Work-In-Progress?node-id=395%3A9946

### Changes
- Added 3 new help pages at `/help`, `/faq`, `/contact`
- Updated some styling on homepage to be consistent with other pages (cards, buttons)
- Removed `/love-assessments` static page & associated code
- Migrated files over from [branding repo](https://github.com/CMSgov/rato-grq-branding) including any existing templates & all static files. _See [this PR](https://github.com/CMSgov/rato-grq-branding/pull/30)_
- Move blueprint styles out of `govready.css` file and into new scss files/scss files that were migrated over from the branding repo

### Testing
- Visit each of `/help`, `/faq`, `/contact` to ensure that the content is consistent with the [Figma designs](https://www.figma.com/file/Q0BD09RJhn8TMdI8BwhiSS/Work-In-Progress?node-id=395%3A9946)
- Ensure that breadcrumbs are functional and reflect the correct links (see Figma)
- Ensure that clicking the cards on the Help page or the card on the FAQ page links to the correct corresponding page/site
- Ensure that clicking the “Help” link in the header and the “Blueprint Support” link in the footer will direct to `/help`
